screens/Map.js

```diff
@@ -21,19 +21,29 @@ export default class Map extends React.Component {
       locationResult: "",
       initialRegion: null
     };
+    this.db = database.ref();
   }
 
   static navigationOptions = {
     header: null
   };
 
   componentDidMount = async () => {
-    this.setState({
-      markers: data
-    });
+    this._listenForUpdatesToDatabase(this.db);
     await this._getLocationAsync();
   };
 
+  _listenForUpdatesToDatabase = db => {
+    db.on("value", snap => {
```

Comment:
does this pull in all the current data and set up a listener so if more data is added, they get updated?

Reply:
Yes, that's correct!